gapii/README.md

```diff
@@ -58,4 +58,14 @@ Some graphics APIs permit simultaneous sharing of contexts by multiple threads.
 GAPII needs to support all of these cases.
 
 Each encoded command holds an identifier of the thread that made the call.
-Commands from all threads are encoded into a single chronological stream.
\ No newline at end of file
+Commands from all threads are encoded into a single chronological stream.
+
+## Command modifications at capture time
+
+Some commands may be need to be modified for the capture to be possible.
```

Comment:
"may be need to be" sounds weird, might be wrong. I would use "may have to be".

Reply:
typo indeed, done